packages/lit-dev-content/site/docs/ssr/authoring.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Authoring components for Lit SSR
+eleventyNavigation:
+  key: Authoring components
+  parent: Server rendering
+  order: 4
+---
+
+{% labs-disclaimer %}
+
+Lit's approach to rendering web components in a server environment places some restrictions on component code to achieve efficient server rendering. When authoring components, keep in mind these considerations to ensure they are compatible with Lit SSR.
+
+Note: The restrictions listed on this page are subject to change as we make improvements to Lit SSR. If you would like to see a certain use case supported, please [file an issue](https://github.com/lit/lit/issues/new/choose) or start a [discussion](https://github.com/lit/lit/discussions) thread.
+
+## Considerations
+
+### Browser only code
+
+Most browser DOM APIs are not available in the Node environment. Lit SSR utilizes a DOM shim that's the bare minimum required for rendering Lit templates and components. For a full list of what APIs are available, see the [DOM Emulation](/docs/ssr/dom-emulation) page.
+
+When authoring components, perform imperative DOM operations from lifecycle methods that are called only on the client, and not on the server. See the [lifecycle](#lifecycle) section below for what specific methods can be used.
```

Comment:
It would be really nice here to include even a high-level example. 

```suggestion
When authoring components, perform imperative DOM operations from lifecycle methods that are called **only on the client**, and not on the server. For example, use `updated()` if you need to measure updated DOM. This callback is only run on the browser, so it's safe to access the DOM.

See the [lifecycle](#lifecycle) section below for a list of which methods are called on the server and which are browser-only.
```

Reply:
You might have a better example, here, just throwing something out.

Reply:
Thank you that's a great example!